src/test/kotlin/net/atos/zac/app/informatieobjecten/InformatieObjectenRESTServiceTest.kt

```diff
@@ -272,5 +343,129 @@ class InformatieObjectenRESTServiceTest : BehaviorSpec() {
                 }
             }
         }
+
+        Given("enkelvoudig informatieobject has been uploaded, and the zaak is open") {
```

Comment:
Not a comment on this test particularly, but the structure of these tests is off. What I mean:
Give = set the current state from where the test will start
When = execute an action on the current state to reach a new state
Then = the new state is verified with expectations

Why don't I see this in the code? All setup and actions are done within the "Then" block... not very clear how state and transition come to the new expected state.

Reply:
Hmm very good point. I never really thought about moving the `given` and `when` code to the right level to be honest. All these tests have been written by me so far. Let me make a PR to refactor this. Good idea. :-)

Reply:
@edgarvonk It seems Given/When/Then and global before block behave a bit strange. We might need a mixture of `beforeEach` and `beforeContainer` to solve the setup/cleanup